client/components/backup-retention-management/retention-options/retention-option-card.tsx

```diff
@@ -63,18 +62,11 @@ const RetentionOptionCard: FunctionComponent< RetentionOptionCardProps > = ( {
 				<div className="space-needed__value">{ spaceNeededText }</div>
 			</div>
 			<div
-				className={ clsx( 'retention-option__upgrade-required', {
+				className={ clsx( 'retention-option__highlight-badge', {
 					'is-visible': upgradeRequired,
 				} ) }
 			>
-				{ translate( 'Upgrade required' ) }
-			</div>
-			<div
-				className={ clsx( 'retention-option__current-plan', {
-					'is-visible': isCurrentPlan,
-				} ) }
-			>
-				{ translate( 'Current plan' ) }
+				<Gridicon icon="star" size={ 12 } /> { translate( 'Upgrade required' ) }
```

Comment:
We might still be using Gridicons in several places on Jetpack Cloud, but let’s try to use [core icons](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library) as much as possible.

I think the `starFilled`could work for this:

```
import { Icon, starFilled } from '@wordpress/icons';
<Icon icon={ starFilled } />
```

![CleanShot 2025-06-24 at 11 57 03@2x](https://github.com/user-attachments/assets/1658dd52-4c7f-404f-a9f8-5926c4cb8505)

Reply:
Thanks for the feedback, Rafa. I've wrote down a reminder to myself to use `Icon` instead of `Gridicon`.